ui/design/src/components/Icon/new-icons/integratin-widget-large.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
typo in the name of this file -> 'integratin-...'

Reply:
Good catch, Thanks. I'll fix that.